superset-frontend/src/explore/components/controls/ConditionalFormattingControl/FormattingPopoverContent.tsx

```diff
@@ -339,6 +370,47 @@ export const FormattingPopoverContent = ({
           </Row>
         )}
       </FormItem>
+      <Row style={{ display: 'flex', height: '40px' }}>
```

Comment:
Do we need the CSS here? We're hoping that using "vanilla" AntD components means we don't need to provide style overrides... but if we do, maybe they should be props for the Row component so that they're used systematically wherever needed?